sdks/python/apache_beam/internal/json_value_test.py

```diff
@@ -67,6 +67,19 @@ def test_with_type(self):
   def test_none_from(self):
     self.assertIsNone(from_json_value(to_json_value(None)))
 
+  def test_long_value(self):
+    self.assertEquals(long(27), from_json_value(to_json_value(long(27))))
+
+  def test_too_long_value(self):
+    try:
+      to_json_value(long(1 << 63))
```

Comment:
In this case I want to test longs that are too long, so it makes sense to have it this way, no?